tests/cli/cli.spec.ts

```diff
@@ -16,32 +16,21 @@ suite(__filename, () => {
 
   test("init and compile", async () => {
     // Initialize a project using the CLI, don't install packages.
-    const initChildProcess = fork(cliEntryPointPath, [
-      "init",
-      "redshift",
-      projectDir,
-      "--skip-install"
-    ]);
-
-    await new Promise(resolve => {
-      initChildProcess.on("close", resolve);
-    });
+    await getProcessResult(
```

Comment:
nit: why not push the executed file and list of args down into `getProcessResult`

Reply:
This is my new thing - why do it that way? At some point I'll probably want to change from execFile to exec, or fork, or something else. ChildProcess is the interface here, it seems less likely I'll have to refactor this in future!